### PR TITLE
Create Team1_week_10-16_10-23.md

### PR DESCRIPTION
**Pull request by apw50**

### CODE REVIEW

File: management/goals/Team1/Team1_week_10-16_10-23.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Manager: Andrew Welby
+
+## Goals
+
+1. AI ship with simple pathfinding movement.
```
Comment (nfarnan) [inline management/goals/Team1/Team1_week_10-16_10-23.md]: This was an unfinished goal from last week, specifically what progress will be made this week?

File: management/goals/Team1/Team1_week_10-16_10-23.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+2. Gravity for Physics
```
Comment (nfarnan) [inline management/goals/Team1/Team1_week_10-16_10-23.md]: This was an unfinished goal from last week, specifically what progress will be made this week?

File: management/goals/Team1/Team1_week_10-16_10-23.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+3. Frame Independent for Physics
```
Comment (nfarnan) [inline management/goals/Team1/Team1_week_10-16_10-23.md]: This was an unfinished goal from last week, specifically what progress will be made this week?

File: management/goals/Team1/Team1_week_10-16_10-23.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+4. Randomized Space Object Tiles for Level Gen.
+5. Procedurally Generated Maze for Level Gen.
+6. Working Movement for Maze for Level Gen.
```
Comment (nfarnan) [inline management/goals/Team1/Team1_week_10-16_10-23.md]: What is going to be moving for the level gen goal?

File: management/goals/Team1/Team1_week_10-16_10-23.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Manager: Andrew Welby
+
+## Goals
+
+1. AI ship with simple pathfinding movement.
+2. Gravity for Physics
+3. Frame Independent for Physics
+4. Randomized Space Object Tiles for Level Gen.
```
Comment (nfarnan) [inline management/goals/Team1/Team1_week_10-16_10-23.md]: Already generating some galaxies sprites in the overlay, what else will be added this week?

**Comment by apw50** [comment]
Hi, I've added the specifics you've requested.

**Pull request merged**